molecule/default/playbook.yml

```diff
@@ -3,6 +3,8 @@
   become: true
   vars:
     pulp_config_secret_key: secret
+    pulp_default_admin_password: ''
+    ansible_python_interpreter: /usr/bin/python3
```

Comment:
This lets the playbook work on distributions like Fedora out of the box. But this breaks the playbook on a distribution like RHEL 7, which ships with Python 2 out of the box. Is there a better place to specify this variable, such as in a dynamically generated `host_vars`, or on the command like when `ansible-playbook` is called?

Note that RHEL 7 has Python 3.4 in the standard libraries, but Pulp 3 requires Python 3.5 or above. As a result, it's not necessary or desirable to install Python 3 on RHEL 7 from the standard libs. When installing Pulp 3 on RHEL 7, it's possible to let `ansible_python_interpreter` be `/usr/bin/python` (which is Python 2), and install and use a Python 3.5 or 3.6 SCL for Pulp 3.

Reply:
I'm trying to solve one problem at a time (having initial green CI) and then adding features (testing against multiple OS which is supported in Molecule). When it comes to RHEL 7 testing, I will figure out a solution for this and other problems (Docker image?).

Reply:
Makes sense.